pupil_src/shared_modules/video_capture/file_backend.py

```diff
@@ -22,21 +22,15 @@
 from camera_models import load_intrinsics
 from .utils import VideoSet
 
-from .base_backend import (
-    Base_Manager,
-    Base_Source,
-    EndofVideoError,
-    Playback_Source,
-)
+from .base_backend import Base_Manager, Base_Source, EndofVideoError, Playback_Source
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 av.logging.set_level(av.logging.ERROR)
 logging.getLogger("libav").setLevel(logging.ERROR)
 
 assert av.__version__ >= "0.4.2", "pyav is out-of-date, please update"
-assert sys.version_info >= (3, 7), \
-    ("We need python 3.7 or later to run Pupil player")
+assert sys.version_info >= (3, 7), "We need python 3.7 or later to run Pupil player"
```

Comment:
I think this line can be removed by now

Reply:
> We do since _fill_gaps() as well as the following code expect a sorted list.

Since 

```
    @property
    def videos(self) -> Sequence[Video]:
        if self._videos is None:
            self._videos = sorted(self.fetch_videos(), key=lambda v: v.path)
        return self._videos
```
self._videos supposed been sorted already, so `loaded_ts = [vid.timestamps for vid in self.videos]` should also sorted by default if I understand it correctly

Reply:
Yes, I did not have this in mind. Please write a specific test that makes sure that the assumption holds true for a given recording.